docs/workflow/requirements/windows-requirements.md

```diff
@@ -30,15 +30,17 @@ git config --system core.longpaths true
 
 ### Visual Studio
 
-Install [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). The Community edition is available free of charge. Visual Studio 2022 17.3 or later is required. Note that Visual Studio and the development tools described below are required, regardless of whether you plan to use the IDE or not. The installation process goes as follows:
+Install [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). The Community edition is available free of charge. Visual Studio 2022 17.8 or later is required. Note that as we ramp up on a given release the libraries code may start using preview language features. While an older IDE may still succeed in building the projects, the IDE may report mismatched diagnostics in the Errors and Warnings window. Using the latest public preview of Visual Studio is required to ensure the IDE experience is well behaved in such scenarios.
+
+Note that Visual Studio and the development tools described below are required, regardless of whether you plan to use the IDE or not. The installation process goes as follows:
 
 * It's recommended to use **Workloads** installation approach. The following are the minimum requirements:
   * **.NET Desktop Development** with all default components,
   * **Desktop Development with C++** with all default components.
 * To build for Arm64, make sure that you have the right architecture-specific compilers installed. In the **Individual components** window, in the **Compilers, build tools, and runtimes** section:
   * For Arm64, check the box for _MSVC v143* VS 2022 C++ ARM64 build tools (Latest)_.
 * To build the tests, you will need some additional components:
-  * **C++/CLI support for v142 build tools (Latest)**.
+  * **C++/CLI support for v143 build tools (Latest)**. Note that this needs to be v14.38 or later in the project properties page
```

Comment:
```suggestion
  * **C++/CLI support for v143 build tools (Latest)**
```
I do not think that this information relevant for setting up the pre-requisites for building the dotnet/runtime repo.